halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/deploy/DeploymentEnvironmentCommand.java

```diff
@@ -15,9 +15,11 @@
  *
  */
 
-package com.netflix.spinnaker.halyard.cli.command.v1.config;
+package com.netflix.spinnaker.halyard.cli.command.v1.config.deploy;
 
 import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.command.v1.config.AbstractConfigCommand;
```

Comment:
Since I moved DeploymentEnvironmentCommand.java to another package, AbstractConfigCommand had to now be imported. DeploymentEnvironmentCommand extends AbstractConfigCommand